src/components/DetailedPodcast/DetailedPodcast.js

```diff
@@ -81,9 +81,9 @@ const Text = styled.p`
   font-weight: 100;
   font-size: 0.8rem;
   grid-column: span 2;
+  color: var(--darker-grey);
```

Comment:
you could name your colors more semantically like `--color-primary`
(this is what Leon suggested when he did a review on my code...)

Reply:
Hi Anni, first of all thank you very much for your review :).
Yes you are right and i will change the naming in the end. But for now im testing around with different colors and therefore i like to have the color name in the variable.